src/basic/capability-util.c

```diff
@@ -18,6 +18,11 @@
 #include "user-util.h"
 
 int have_effective_cap(int value) {
+#if defined(FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION) && defined(HAS_FEATURE_MEMORY_SANITIZER)
+        /* OSSFuzz does not have libcap built with the memory sanitizer, and reports many false-positive
+         * about the function, like issues #25808, #25827, and #25840. See issue #20542 for more details. */
+        return -EOPNOTSUPP;
```

Comment:
hmm, can we add something like `#pragma message` or so for cases like this? otherwise this is so hard to notice later on.

i guess the whole comment should just be a pragma message.